src/components/DayPicker.jsx

```diff
@@ -68,11 +72,17 @@ const defaultProps = {
   monthFormat: 'MMMM YYYY',
 };
 
+const triggerInitialVisibleMonth = (props) => (
+  props.hidden ? moment() : props.initialVisibleMonth()
```

Comment:
Since it's used in two separate components, my hope was that we'd define it in one place and import it in both.